scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -216,4 +216,5 @@
   </div>"
   title="">{{ variant.str_repid }}
   </a>
+  {{ variant.stripy_link }}
```

Comment:
Where does this change reflect on the STR variantS page?

Reply:
Ahem, thank you - not sure it does! It was supposed to be in the variantS popover for the locus name, but I left it unfinished when deciding to add it to the variant page directly. Will fix!

Reply:
Oki, now it looks like this:
<img width="492" alt="Screenshot 2022-02-28 at 15 40 03" src="https://user-images.githubusercontent.com/758570/156002366-b20a55c9-37a0-45be-8ff3-8ced447e6b4e.png">
with the S-badge link going to STRipy and the G-one to GnomAD. And OMIM disease info is also in the popover.